src/main/python/rlbot/setup_manager.py

```diff
@@ -575,6 +582,53 @@ def kill_matchcomms_server(self):
             self.matchcomms_server.close()
             self.matchcomms_server = None
 
+    def mergeTASystemSettings(self):
```

Comment:
It seems like you could move this logic into its own file pretty easily, is that true? I wish setup_manager.py was a smaller file.

Reply:
Done this as of 43ab9b1